pkg/controller/pxc/tls.go

```diff
@@ -180,7 +210,26 @@ func (r *ReconcilePerconaXtraDBCluster) createIssuer(ownRef []metav1.OwnerRefere
 		},
 	})
 	if err != nil && !k8serr.IsAlreadyExists(err) {
-		return fmt.Errorf("create issuer: %v", err)
+		return fmt.Errorf("create selfsigned issuer: %v", err)
+	}
+	return nil
+}
+
+func (r *ReconcilePerconaXtraDBCluster) createIssuer(ownRef []metav1.OwnerReference, namespace, issuer string, caCertSecret string) error {
+	err := r.client.Create(context.TODO(), &cm.Issuer{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:            issuer,
+			Namespace:       namespace,
+			OwnerReferences: ownRef,
+		},
+		Spec: cm.IssuerSpec{
+			IssuerConfig: cm.IssuerConfig{
+				CA: &cm.CAIssuer{SecretName: caCertSecret},
```

Comment:
Do we really need two functions with only differences in one line, which can be passed as an argument?